src/shared/discover-image.c

```diff
@@ -58,11 +58,13 @@ static const char* const image_search_path[_IMAGE_CLASS_MAX] = {
                             "/usr/local/lib/portables\0"
                             "/usr/lib/portables\0",
 
+        /* Note that we don't allow storing extensions under /usr/lib, unlike with other image types. That's
+         * because extension images are supposed to extend /usr, so you get into recursive races, especially
```

Comment:
doesn't matter, but as per coding style we like it if in docs/comments/log messages dirs are suffixed with "/" to indicate that they are dirs. i.e. /usr → /usr/